src/main/resources/templates/macros/json/output.vm

```diff
@@ -1,16 +1,21 @@
-#macro(includeOutput, $output)
+#macro(includeOutput, $outputs, $isPassed)
 
-#if ($output)
+#if($outputs)
   <div class="outputs inner-level">
-    #foreach($message in $output.getMessages())
+    #foreach($output in $outputs)
```

Comment:
Sure. Here is demo screenshots.
Current output:
![old](https://cloud.githubusercontent.com/assets/2842503/22456431/d0613fa4-e7ab-11e6-964b-dc7e21cf3f71.png)

New output:
![new](https://cloud.githubusercontent.com/assets/2842503/22456438/da019d88-e7ab-11e6-96d1-8214ccb71143.png)